tensorflow/lite/micro/tools/ci_build/test_hexagon.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+set -e
+
+# Default prebulit core library on docker
+HEXAGON_TFLM_LIB=/root/Qualcomm/hexagon_tflm_core.a
+
+if [[ $# -ge 1 ]];
+then
+  HEXAGON_TFLM_LIB=$1
+fi
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ROOT_DIR=${SCRIPT_DIR}/../../../../..
+cd "${ROOT_DIR}"
+pwd
+
+source tensorflow/lite/micro/tools/ci_build/helper_functions.sh
+
+readable_run make -f tensorflow/lite/micro/tools/make/Makefile clean
+
+# TODO(b/143904317): downloading first to allow for parallel builds.
+readable_run make -f tensorflow/lite/micro/tools/make/Makefile third_party_downloads
+
+readable_run make -f tensorflow/lite/micro/tools/make/Makefile \
+  TARGET=hexagon \
+  OPTIMIZED_KERNEL_DIR=hexagon \
+  OPTIMIZED_KERNEL_DIR_PREFIX=third_party \
+  HEXAGON_TFLM_LIB=${HEXAGON_TFLM_LIB} \
+  build -j$(nproc)
+
```

Comment:
let's add a TODO(b/197888845): and a commented out line with make test?

Reply:
Done. Thanks for the suggestions!